ContinuousIntegrationAndDeployment.md

```diff
@@ -40,4 +40,102 @@ On every successful build, continuous delivery is kicked off. Our reference arch
 
 `Canary` releases are deployed to the cluster on each successful build. This will ensure that changes are constantly delivered and high deployment velocity is available.
 
-`Stable` releases are typically a couple versions behind `canary` and the images are deemed stable for production. This will ensure that changes are highly available. Our solution requires an administrator to sign off on `stable` deployments. Rollbacks are easily accessible since each build is versioned.
\ No newline at end of file
+`Stable` releases are typically a couple versions behind `canary` and the images are deemed stable for production. This will ensure that changes are highly available. Our solution requires an administrator to sign off on `stable` deployments. Rollbacks are easily accessible since each build is versioned.
+
+## Creating Continuous Integration
+
+Each Repository requires 2 Build Definitions: one for PRs and one for master builds.
+
+**Prerequisites**
+- Kube Config must be on build/release machines.
```

Comment:
How do you get kube config on the build/release machines in VSTS? Would be good to detail this one out.